client/src/pages/private/SiteParticipantsTable.js

```diff
@@ -247,17 +247,22 @@ export default ({ id, siteId, onArchiveParticipantAction }) => {
     };
 
     fetchParticipants();
-  }, [siteId, setRows, setFetchedRows, setFetchedWithdrawnRows, setLoadingData]);
+  }, [id, setRows, setFetchedRows, setFetchedWithdrawnRows, setLoadingData]);
```

Comment:
I think this siteId vs id naming was deliberate because site entities have both keys in the DB... this change could introduce some confusion, we might want to revert to the previous name in a subsequent PR

Reply:
unless this WAS using the id and not the siteId, in which case this is a good fix!

Reply:
I think the key thing is that id != siteId

So I just want to make sure we're still functional and using the correct term in the code, to reduce future confusion about this...

Reply:
I removed siteId from the component to prevent confusion. I would rename the variable as well, but because it's a prop this really is an issue with the parent component, and I don't want to tack a rework on to this PR, it's big enough.

Reply:
Sounds good to me - if there's still cleanup that would make sense to do, let's track as tech debt